NSS-Day-Cohort-59/ares-army-tabloidfullstack
# Added View All Posts and View Post Details functionality

closes #43 
closes #45 

Created PostRepository.cs, IPostRepository.cs, and PostController.cs. 
Added methods to get all posts and get posts by Id.
Created postManager.js, PostList.js, and PostDetails.js. 
Added functionality to view all posts and view post details by id.

To test, log into the app,
Click on Post Management. 
You should see the list of all posts (will go back to display category names when the category class is complete)
Click on a post title
You should be taken to the details page of the post
